src/test/resources/examples/operationsSchema/models/Models.kt

```diff
@@ -0,0 +1,85 @@
+package examples.operationsSchema.models
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonValue
+import javax.validation.Valid
+import kotlin.Int
+import kotlin.String
+import kotlin.collections.Map
+
+enum class EnumPostParametersP(
+    @JsonValue
+    val value: String
+) {
+    X("X"),
+
+    Y("Y");
+
+    companion object {
+        private val mapping: Map<String, EnumPostParametersP> =
```

Comment:
**praise (non-blocking):** This is for parsing? I like it.

Reply:
Sorry, I don't get your question.

Reply:
Just a praise :)

Reply:
I'm not sure what I've done to deserve it but thanks :D